core/src/storage/device.py

```diff
@@ -33,6 +33,7 @@
 _SLIP39_IDENTIFIER         = const(0x10)  # bool
 _SLIP39_ITERATION_EXPONENT = const(0x11)  # int
 _SD_SALT_AUTH_KEY          = const(0x12)  # bytes
+INITIALIZED               = const(0x13)  # bool (0x01 or empty)
```

Comment:
nit: whitespace

Reply:
Thanks, fixed in afd8acb25f31dc8d8cedc15576f3b37262f7a455.